glean-core/android/src/main/java/mozilla/telemetry/glean/rust/LibGleanFFI.kt

```diff
@@ -354,16 +353,16 @@ internal interface LibGleanFFI : Library {
 
     fun glean_destroy_timing_distribution_metric(handle: Long)
 
-    fun glean_timing_distribution_set_start(metric_id: Long, start_time: Long): GleanTimerId
+    fun glean_timing_distribution_set_start(metric_id: Long, start_time: Long): Long
```

Comment:
I guess this isn't terrible to just be a `Long` here since it should only be our platform glue code calling it and converting it to the GleanTimerId, although I wonder if there isn't some way to use Kotlin magic to keep the return type?  Not a blocker, just thinking out loud...

Reply:
You're right! I tried to think about that, but nothing came to mind :(